src/Uno.UI/UI/Xaml/Media/Animation/Animators/AnimatorFactory.Android.cs

```diff
@@ -332,10 +332,57 @@ private static ValueAnimator GetRelativeAnimator(Java.Lang.Object target, string
 		}
 
 		/// <summary>
-		/// Ensures that scale value is without the android accepted values
+		/// Ensures that scale value is within the android accepted values
 		/// </summary>
 		private static double ToNativeScale(double value)
 			=> double.IsNaN(value) ? 1 : value;
 
+		internal static bool TryGetNativeAnimatedValue(Timeline timeline, out object value)
+		{
+			value = null;
+
+			if (timeline.GetIsDependantAnimation() || timeline.GetIsDurationZero())
+			{
+				return false;
+			}
+
+			var info = timeline.PropertyInfo.GetPathItems().Last();
+			var target = info.DataContext;
+			var property = info.PropertyName
+				.Split(new[] { '.' }).Last()
+				.Replace("(", "").Replace(")", "");
```

Comment:
There's already a parser for that if I'm not mistaken, we probably should be reusing it.

Reply:
Didn't find any, so I added one.